### PR TITLE
add VAD prevalence and RR tables

### PR DESCRIPTION
**Pull request by yaqi7**

### CODE REVIEW

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst [hunk 1]
```diff
@@ -209,6 +209,23 @@ The causes affected by the Vitamin A Deficiency risk in GBD 2017 include
     -
     - 3.65 (2.23 - 5.97)
     - No (only one study)
+	
+.. list-table:: Table 4.2: Pulled relative risks of GBD 2017 from GBD mapping 
+  :widths: 15 14 14 
+  :header-rows: 1
+
+  * - Cause
```
Comment (collijk) [inline docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: In the same way we have the tables for the cause model, we should note where these relative risks come from at the draw level.  Risks are easy.  rrs come from the `relative_risk` source pulled with the rei_id.

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst [hunk 2]
```diff
@@ -310,6 +327,23 @@ approach is twofold:
     provides a way to directly model these effect sizes in a way that preserves
     this interpretation.
 
+.. list-table:: Prevalence of Vitamin A deficiency in GBD 2017
+  :widths: 15 14 14 
+  :header-rows: 1
+
+  * - Source
```
Comment (collijk) [inline docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: There is no actual data in the gbd mapping, just metadata (like ids and distribution type).  GBD compare is a good place to look, but for vitamin a, there are two potential draw level sources we might use: prevalence from `como` using cause id 389 or exposure for the `exposure` source using rei_id 96.

Comment (collijk) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: Currently it is unclear what's in this table.

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst (hunk 1 above)
Comment (yaqi7) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: I'll add the data source note. Thanks

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst (hunk 2 above)
Comment (alibow) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: Agreed, we should clarify where the data in the second column is coming from!

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst (hunk 1 above)
Comment (alibow) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: I am confused about the difference between table 4.1 and 4.2. What are the different data sources and which one should we use? Do we know why they are different?

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst (hunk 2 above)
Comment (yaqi7) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: This table aims to compare prevalence data from two sources (como using cause id 389 and GBD compare). They are both from Como. I will replace the GBD compare data with the exposure source using rei_id 96.

File: docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst (hunk 1 above)
Comment (yaqi7) [reply docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst]: Good question. Table 4.1 is from appendix of GBD lancet paper, table 4.2 are pulled data from Como

**Comment by yaqi7** [comment]
> are the differences because you are comparing different age groups?

@aflaxman Not only due to different age groups, draws data pulled from exposure source using rei_id 96 are DisMod model results which are not necessarily consistent with GBD results.

**Comment by yaqi7** [comment]
> are the differences because you are comparing different age groups?

For pulled same age group (1 - 5) VAD prevalence data, these two sources data are consistent. However, RR data of GBD appendix is different from that of pulled data even though they both are in age group 1 - 5 year.

**Comment by yaqi7** [comment]
Updated tables:
https://github.com/ihmeuw/vivarium_research/blob/e2bcc326f278ac716a193c4bb1bfae0c39be8e42/docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

Calculation Notebook: https://github.com/ihmeuw/vivarium_data_analysis/blob/feature/vitA_deficiency/pre_processing/vitamin_a_deficiency/VAD_prevalence_RR.ipynb
Please let me know if I need further revise it @collijk

**Pull request merged**